src/pages/index.js

```diff
@@ -1,91 +1,180 @@
 import React from 'react';
-import { Redirect } from 'react-router';
 import clsx from 'clsx';
 import Layout from '@theme/Layout';
 import Link from '@docusaurus/Link';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import styles from './styles.module.css';
+import { ResponsivePlayer } from '../components'
 
 const features = [
   {
-    title: <>Easy to Use</>,
-    imageUrl: 'img/undraw_docusaurus_mountain.svg',
+    title: <>Server management</>,
+    imageUrl: 'img/server.png',
     description: (
       <>
-        Docusaurus was designed from the ground up to be easily installed and
-        used to get your website up and running quickly.
+        Deploy and manage your own instance of the Temporal Server.
       </>
     ),
+    goto: '/docs/server-introduction',
   },
   {
-    title: <>Focus on What Matters</>,
-    imageUrl: 'img/undraw_docusaurus_tree.svg',
+    title: <>Application development</>,
+    imageUrl: 'img/layers.png',
     description: (
       <>
-        Docusaurus lets you focus on your docs, and we&apos;ll do the chores. Go
-        ahead and move your docs into the <code>docs</code> directory.
+        Start building invincible applications using an SDK in the language of your choice!
```

Comment:
suggestion (take it or leave it) - less marketingey, just get to the point, mention the languages

```suggestion
        Learn Temporal concepts and look up references in Go, Java, PHP, or other SDKs
```